Styles/styles.css

```diff
@@ -12,26 +15,561 @@
     5. Media Queries
 ////////////////////////*/
 
-*{
+:root {
+    /* Colors */
+    --color-purple: #5722CD;
+    --color-blue: #4845E4;
+    --color-pink: #ED4385;
+    --color-black: #0C0C0C;
+    --color-white: #F2F0FF;
+    --color-lightgray: #F3F6F9;
+    --color-gray: #7A7A7A;
+}
+
+* {
     margin: 0;
     padding: 0;
     box-sizing: border-box;
 }
 
+body {
+    font-family: 'Merriweather', sans-serif;
+    font-size: 16px;
+    color: var(--color-black);
+}
+
+
 /* ------------------------------- 1. Global Styles -------------------------------- */
 
 /* ------------------------------- 2. Typography ----------------------------------- */
 
 /* ------------------------------- 3. Layout --------------------------------------- */
 
 /* --------------- 3.1 Reusables ----------------------- */
+.outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    /*border: 2px blue dashed;*/
+    /*align-items: center;
+    padding: 2rem;*/
+}
+
+.inner-container {
+    display: flex;
+    flex-basis: 1200px;
+    flex-direction: column;
+    justify-content: center;
+  /*  border: dashed red 2px;*/
+    align-items: center;
+
+}
 
 /* --------------- 3.2 Other --------------------------- */
 
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
+.nav-section {
+    background-color: #4B49E8;
+
+}
+
+.nav-inner {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    font-family: 'Roboto', sans-serif;
+    color: white;
+}
+
+.navbuttons {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    width: 100%;
+    list-style: none;
+    gap: 30px;
+    height: 50px;
+    margin: 0 20px 0 0;
+}
+
+.header-section {
+    background-color: #4B49E8;
+    color: white;
+}
+
+.header-sectionin {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 50px
+}
+
+.header-sectionin h1 {
+    font-weight: 400;
+    text-align: center;
+    margin-bottom: 20px;
```

Comment:
Bij regel 95 geef je aan margin 4 waardes en hier maar 1. Beide werken natuurlijk zoals verwacht, maar het is belangrijk om consistent te zijn in je code. Dus gebruik of het 1, of het ander.